### PR TITLE
Fixes #699 - Update test_hosted_paid_app_submission to work with the different forms that exist on dev and staging

### LINKED ISSUE #699: Update test_hosted_paid_app_submission to work with the different forms that exist on dev and staging
https://webqa-ci.mozilla.com/view/Marketplace/job/marketplace.stage.developer_hub.saucelabs/ is failing because there are different forms that appear when creating a payment account on dev and staging. The test passes on dev but fails on staging.

We need to update the test and/or page objects to allow it to pass in both environments.

### PR DESCRIPTION
**Pull request by bobsilverberg**
This addresses issue #699, which should fix the failure of `test_hosted_paid_app_submission` as seen at https://webqa-ci.mozilla.com/view/Marketplace/job/marketplace.stage.developer_hub.saucelabs/308/testReport/

@davehunt r?

### CODE REVIEW

File: mocks/mock_bango_payment_account.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
```
Comment (davehunt) [inline mocks/mock_bango_payment_account.py]: Nit: Remove the shebang, it's not needed for files that are not executed directly.

**Comment by davehunt** [comment]
Is the difference between dev and staging permanent, or will it go away in a pending deploy? Would it be considered a failure if Bango appeared on both environments? If so, we might want to set an expectation based on the environment under test.

The code changes look okay to me, however I would prefer to abstract the payment account details into a separate objects. Perhaps BangoPaymentAccount and ReferencePaymentAccount.

File: mocks/mock_bango_payment_account.py (hunk 1 above)
Comment (bobsilverberg) [inline mocks/mock_bango_payment_account.py]: Done

**Comment by bobsilverberg** [comment]
Thanks @davehunt. I have addressed your comments. r?

**Comment by davehunt** [comment]
r+ I like this a lot more! Thanks!

**Comment by bobsilverberg** [comment]
Thanks @davehunt!

**Pull request merged**